assets/in

```diff
@@ -16,26 +18,42 @@ fi
 
 # parse incoming config data
 payload=$(cat)
-bucket=$(echo "${payload}" | jq -r '.source.bucket')
+
+bucket="$(echo "${payload}" | jq -r '.source.bucket')"
+if [ -z "${bucket}" ]; then
+  echo "No bucket supplied"
+  exit 1
+fi
+
 endpoint_url="$(echo "${payload}" | jq -r '.source.endpoint // ""')"
 region="$(echo "${payload}" | jq -r '.source.region // "us-east-1"')"
 use_v4="$(echo "${payload}" | jq -r '.source.use_v4 // ""')"
-path=$(echo "${payload}" | jq -r '.source.path // ""')
-suffix=$(echo "${payload}" | jq -r '.source.suffix // ""')
+path="$(echo "${payload}" | jq -r '.source.path // ""')"
```

Comment:
If you provide a random string path that doesn't exist the script finishes successfully. IMHO it should fail